### ISSUE TITLE
Embed nullish-coalesce feature report (MDN)

### ISSUE THREAD

**Issue by SphinxKnight** [bocoup/test262-report-issue-tracker#18]
### Summary
TL;DR: I didn't manage to build a https://test262.report/embed/features/ URL for the `coalesce-expression` feature to embed it on MDN.

Since the nullish coalescing expression landed on Firefox Nighly, I started writing the page for this operator on MDN.
It seems that the feature is named `coalesce-expression` in the [test262 repo](https://github.com/tc39/test262/blob/master/features.txt). However, after having tried different paths under https://test262.report/embed/features/, I think that there is something missing for `coalesce-expression` to be published under `/features/` and I cannot pinpoint what exactly causes that.

### Expected behavior

Being able to embed a report by using a URL like https://test262.report/embed/features/coalesce-expression or https://test262.report/embed/features/nullish-coalesce in the same fashion that https://test262.report/embed/features/optional-chaining

### Actual behavior

I did not find such a URL to embed the report for nullish coalescing or the URL is missing/needs some code to be usable.

### Steps to reproduce

* Go to https://test262.report/embed/features
* Do not find any feature with "coalesce"
* Check https://github.com/tc39/test262/blob/master/features.txt where `coalesce-expression` is provided

### Notes

I know that the https://github.com/mdn/kumascript/blob/master/macros/EmbedTest262ReportResultsTable.ejs macro on MDN is not widespread and that its use may be experimental. Please discard this issue if the fix/solution is not worth the effort in this experimentation :)

**Comment by leobalter** [comment]
thanks for the report @SphinxKnight! We deployed some updates that now show https://test262.report/embed/features/coalesce-expression

Please let me know if that works for you

**Comment by SphinxKnight** [comment]
Works like a charm :) Thanks for the swift action!
![image](https://user-images.githubusercontent.com/2413436/69122889-3d6fbe80-0aa0-11ea-883b-86f590dce7f0.png)

**Comment by leobalter** [comment]
Thank you for filing the issue!
